hack/arktos-up-scale-out-poc.sh

```diff
@@ -55,12 +55,7 @@ fi
 KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 
 # for POC, the kubelet_flags is used for the new temporary kubelet commandline args
-KUBELET_FLAGS="--tenant-servers="${TENANT_SERVERS}
-# --tenant-servers set the tenant server urls for node-lifecycle controller
-CONTROLLER_MANAGER_EXTRA_FLAGS="--tenant-servers="${TENANT_SERVERS}
-
-echo KUBELET_FLAGS for new kubelet commandline --tenant-servers
-echo ${KUBELET_FLAGS}
+KUBELET_FLAGS="--tenant-servers=http://172.30.0.148:8080,http://172.30.0.24:8080"
```

Comment:
with hardcoded IPs for tenant servers ?

Reply:
I had to hardcode it in my local test as I wanted to let kubelet and KCM share same environment variable TENANT_SERVERS. But kubelet code was not updated to use kubeconfig so I did not update the corresponding script.

Will fix the hardcode servers in this CR